src/features/currentTodo.ts

```diff
@@ -1,10 +1,59 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { User } from '../types/User';
 
-const initialState = null as Todo | null;
+export interface CurrentTodo {
+  currentTodo: Todo | null;
+  user: User | null;
+  curentLoading: boolean;
+}
 
-export const currentTodoSlice = createSlice({
+const initialState: CurrentTodo = {
+  currentTodo: null,
+  user: null,
+  curentLoading: false,
+};
+
+const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    fetchCurrentStart(state) {
+      const theState = state;
```

Comment:
FYI: in files with redux slices you can just disable the lint rule instead of doing this